src/components/InstancesList/InstancesList.js

```diff
@@ -765,6 +763,16 @@ class InstancesList extends React.Component {
           query: row.name,
         });
         break;
+      case browseModeOptions.CONTRIBUTORS:
```

Comment:
It seems that this case label duplicates the one above. Is it expected?

Reply:
You are right, Thanks, I fixed it!